Making the licensing more precise (#2)
Louka3000/OpenVR-Dynamic-Resolution: README.md

## Settings Descriptions

> Settings are found in the `settings.ini` file. Do not rename that file. It should be located with your executable file (e.g., `OpenVR-Dynamic-Resolution.exe`).

Reviewer: ```suggestion
Settings are found in the `settings.ini` file. Do not rename that file. It should be located in the same folder as your executable file (`OpenVR-Dynamic-Resolution.exe`).
```
User shouldn’t change the name of the executable, or else auto-start with SteamVR won’t work
